draft-irtf-cfrg-vdaf.md

```diff
@@ -610,7 +619,7 @@ security considerations for DAFs and VDAFs.
 * Prio3: Remove an intermediate PRG evaluation from query randomness generation.
   (\*)
 
-* Add additional guidance for choosing FFT-friendly fields.
+* Add additional guidance for choosing NTT-friendly fields.
```

Comment:
Should we leave this as-is since it's part of the changelog, or is this fine?

Reply:
Good catch. it depends on whether we mention renaming FFT to NTT in the change log: I decided to, so I think we should here. Fixed.